cpp/src/parquet/metadata_test.cc

```diff
@@ -209,7 +208,7 @@ TEST(Metadata, TestBuildAccess) {
     ASSERT_EQ(10, rg2_column1->data_page_offset());
     ASSERT_EQ(26, rg2_column2->data_page_offset());
     ASSERT_EQ(2, rg2_column1->encoding_stats().size());
-    ASSERT_EQ(2, rg2_column2->encoding_stats().size());
+    ASSERT_EQ(3, rg2_column2->encoding_stats().size());
```

Comment:
Can we also check the actual encodings values in this test?

Reply:
Sure, will check it tomorrow